charts/digibyte/README.md

```diff
@@ -1,3 +1,35 @@
-# Digibyte Node
+## Digibyte Full Node
+
+This helm chart(solution) is going to launch a Digibyte full node on Mainnet. The chart should have 3 inputs,
+
+#### Inputs
+
+* Node Name (which will serve as the subdomain or domain name)
+* rpcuser (The node RPC credentials user/pass)
+* rpcpasswd
+
+Here is an example,
+
+```
+git clone https://github.com/threefoldtech/vdc-solutions-charts.git
+cd vdc-solutions-charts
+helm --install dgb-node charts/digibyte --set global.ingress.host="mynode" --set env.rpcuser="abdulgig" --set env.rpcpasswd="test7654" 
```

Comment:
shouldn't `global.ingress.host` be a fully qualified domain name to allow access to the service? The ingress resource uses it as a domain name to filter requests and also to generate certificate so how mynode work as a host?

Reply:
The node name should be a sub-domain. You can override that and the append the node name with gateway to work as an FQDN from the chatflow itself. I will update the README for clarity

Here is an example ,
http://masid-cryptpad-test1.webg1test.grid.tf/